openupgrade_scripts/scripts/pos_restaurant/17.0.1.0/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,46 @@
+---Models in module 'pos_restaurant'---
+obsolete model restaurant.printer (renamed to pos.printer in module point_of_sale)
+# NOTHING TO DO: handled in the point_of_sale module migration scripts
+
+---Fields in module 'pos_restaurant'---
+pos_restaurant / pos.config               / is_table_management (boolean) : DEL
+# NOTHING TO DO: Odoo dropped these fields in https://github.com/odoo/odoo/commit/e3ad1eb058db6329b7f6b8e4880e3610eee22dd9
+
+pos_restaurant / pos.config               / module_pos_restaurant_appointment (boolean): NEW
+# NOTHING TO DO: new feature used to install the pos_restaurant_appointment module in Enterprise
+
+pos_restaurant / pos.order                / multiprint_resume (char)      : DEL
```

Comment:
This field has been renamed into:

https://github.com/odoo/odoo/commit/73aae73f1ba4ad9471d0cded3d8f02cac8cc3669#diff-29cbaebb5b63b539ab173d9340b2aec87b9ad63cd322eec2347879c5412bd50bR241